.github/workflows/test.yaml

```diff
@@ -41,3 +41,10 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: clippy
+
+      - name: Run BPF conformance tests
+        run: |
+          docker run -v ${{github.workspace}}:/rbpf --rm ghcr.io/alan-jowett/bpf_conformance:main \
```

Comment:
Can we pin to a specific version of the container instead of `main`? I understand this means we have to update here regularly, but otherwise the CI may suddenly break without apparent changes in rbpf.

Reply:
Although looking at it again, I see we've got this `|| true` at the end at the moment, so maybe we can keep `main` until we're compliant enough to pass the tests and raise actual failures

Reply:
Pinned.